build/calc_release_version.py

```diff
@@ -100,26 +102,55 @@ def check_head_tag():
 
     return None
 
+def get_next_minor (prerelease_marker):
+    """
+    get_next_minor does the following:
+    Inspect the branches that fit the convention for a release branch.
+    Choose the latest increment the minor version. Append .0 to form the new version (e.g., r1.21 becomes 1.22.0)
```

Comment:
The sentence "Choose the latest increment the minor version." does not make sense.  Is it meant to be something like "Choose the latest minor release and increment the minor version."?  Also, the "Append .0 ..." sentence looks like it belongs on its own line.

Reply:
Updated.